cmd/releasego/add-retry-instructions.go

```diff
@@ -131,3 +100,69 @@ func handleAddRetryInstructions(p subcmd.ParseFunc) error {
 
 	return nil
 }
+
+// Match env vars like "poll1buildId=123" with the number and value in groups. If there are
+// multiple "=", treat the leftmost one as the separator. Ignore case.
+var reg = regexp.MustCompile(`(?i)poll(\d+)(.+?)=(.*)`)
```

Comment:
What is `(?`?

Reply:
That sets flags: https://github.com/google/re2/wiki/Syntax `(?flags)` in "Grouping" table, `i` being case-insensitive.